SIFT.py

```diff
@@ -166,6 +166,15 @@ def _find_harris_interest_points(self, image_bw: np.ndarray, k: int, feature_wid
         c = c[indices]
 
         return x, y, c
+    
+    def compute_dominant_orientation(self, magnitudes, orientations):
```

Comment:
Use `_compute_dominant_orientation` as name instead to suggest that this is a private function. Also move down to `ScaleRotInvSIFT`